docs/content/guides/developer/app-examples/trustless-swap.mdx

```diff
@@ -1048,117 +1222,95 @@ Note that `dApp Kit` provides a set of hooks for making query and mutation calls
 
 :::
 
-### Connect wallet
+Create `src/components/Header.tsx`. This file contains the navigation links and the connect wallet button:
 
-The dApp Kit comes with a pre-built React.js component called `ConnectButton` displaying a button to connect and disconnect a wallet. The connecting and disconnecting wallet logic is handled seamlessly so you don't need to worry about repeating yourself doing the same logic all over again.
+```tsx
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
 
-Place the `ConnectButton` in the header:
+import { ConnectButton } from "@mysten/dapp-kit";
+import { SizeIcon } from "@radix-ui/react-icons";
+import { Box, Container, Flex, Heading } from "@radix-ui/themes";
+import { NavLink } from "react-router-dom";
 
-```ts title='src/components/Header.tsx'
-import { ConnectButton } from '@mysten/dapp-kit';
-import { Box, Button, Container, Flex, Heading } from '@radix-ui/themes';
+const menu = [
+  {
+    title: "Escrows",
+    link: "/escrows",
+  },
+  {
+    title: "Manage Objects",
+    link: "/locked",
+  },
+];
 
 export function Header() {
-	return (
-		<Container>
-			<Box className="connect-wallet-wrapper">
-				<ConnectButton />
-			</Box>
-		</Container>
-	);
+  return (
+    <Container>
+      <Flex
+        position="sticky"
+        px="4"
+        py="2"
+        justify="between"
+        className="border-b flex flex-wrap"
+      >
+        <Box>
+          <Heading className="flex items-center gap-3">
+            <SizeIcon width={24} height={24} />
+            Trading Demo
+          </Heading>
+        </Box>
+
+        <Box className="flex gap-5 items-center">
+          {menu.map((item) => (
+            <NavLink
+              key={item.link}
+              to={item.link}
+              className={({ isActive, isPending }) =>
+                `cursor-pointer flex items-center gap-2 ${
+                  isPending
+                    ? "pending"
+                    : isActive
+                      ? "font-bold text-blue-600"
+                      : ""
+                }`
+              }
+            >
+              {item.title}
+            </NavLink>
+          ))}
+        </Box>
+
+        <Box className="connect-wallet-wrapper">
+          <ConnectButton />
+        </Box>
+      </Flex>
+    </Container>
+  );
 }
 ```
 
+The dApp Kit comes with a pre-built React.js component called `ConnectButton` displaying a button to connect and disconnect a wallet. The connecting and disconnecting wallet logic is handled seamlessly so you don't need to worry about repeating yourself doing the same logic all over again.
```

Comment:
minor nit (and genuine question): Do we say 'The dApp Kit', or just 'dApp kit'?

Reply:
@ronny-mysten What do you think here?